packages/cli/src/commands/identity/test-domain-claim.ts

```diff
@@ -0,0 +1,49 @@
+import { createDomainClaim } from '@celo/contractkit/lib/identity/claims/claim'
+import { verifyDomainRecord } from '@celo/contractkit/lib/identity/claims/verify'
+import { flags } from '@oclif/command'
+import chalk from 'chalk'
+import { ClaimCommand } from '../../utils/identity'
+
+export default class TestDomainClaim extends ClaimCommand {
```

Comment:
Could we reuse our existing `get-metadata` and `show-metadata` commands?

Reply:
ping on this

Reply:
Integrated in those commands:
```
bin/run account:show-metadata  ~/Projects/conf/Celo/orco-dev-meta.json
Metadata at /home/aitor/Projects/conf/Celo/orco-dev-meta.json contains the following claims: 

TXT Record celo-site-verification found
Signature verified successfully
Type   Value            Status    Created At 
DOMAIN Domain: orco.dev Verified! 3 days ago 
```